azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/features/interfaces/ISupportFilesFeature.kt

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.features.interfaces
+
+import android.content.Context
+import com.azure.android.communication.calling.CallClient
+import com.azure.android.communication.ui.calling.features.ACSFeature
+import java.io.File
+
+abstract class ISupportFilesFeature : ACSFeature {
```

Comment:
It reminds me a C# code style with prefixing `I` for interfaces.
Should we rather use `SupportFilesFeature` for interface and `SupportFilesFeatureimpl` for class.
https://kotlinlang.org/docs/coding-conventions.html#function-names

Reply:
Will update.